docs/scripts/add_version.py

```diff
@@ -0,0 +1,91 @@
+"""Utilities for adding versions when building the documentation."""
+
+import sys
+from pathlib import Path
+
+
+def add_version_to_selector_page(version: str) -> None:
+    """Add the newly built version to the version selection overview.
+
+    Args:
+        version: The version that should be added.
+    """
+    indent = "        "
+    new_line = (
+        f"{indent}<li><a href="
+        f'"https://avhopp.github.io/baybe_dev/{version}/">{version}'
+        "</a></li>\n"
+    )
+    file = Path("versions.html")
+    modified_lines = []
+    with file.open(mode="r") as f:
+        lines = f.readlines()
+        for line in lines:
+            modified_lines.append(line)
+            # Add new line at correct position which is in the first line after stable
+            if "Stable" in line:
+                modified_lines.append(new_line)
+    with file.open(mode="w") as f:
+        f.writelines(modified_lines)
+
+
+def adjust_version_to_html(version: str) -> None:
```

Comment:
is it possible to add the version as its own section? What i mean is instead of at the very bottom and under the `Package` section it could be at the very top like this
![image](https://github.com/user-attachments/assets/bee347c6-760e-46ff-9e3a-020f67110e27)
where `General` would say `Version` and `Readme` would say `0.9.1` or `latest` etc

Reply:
nice, would remove the `s` in the end because it only always shows one version, resolving